components/TextFields/tests/unit/OutlinedTextFieldColorThemerTests.m

```diff
@@ -154,4 +155,39 @@ - (void)_disabled_testThemerWithBaselineColorSchemeAppliesToMultilineTextFieldDi
   XCTAssertEqualObjects(textField.underline.disabledColor, controller.disabledColor);
 }
 
+- (void)testOutlinedTextFieldBorderStrokeColorTheming {
+  // Given
+  MDCTextField *textField = [[MDCTextField alloc] init];
+  MDCTextInputControllerOutlined *controller =
+      [[MDCTextInputControllerOutlined alloc] initWithTextInput:textField];
+  MDCSemanticColorScheme *colorScheme =
+      [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+
+  // When
+  controller.borderStrokeColor = UIColor.redColor;
+  [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme
+                                      toTextInputController:controller];
+
+  // Then
+  XCTAssertEqualObjects(textField.borderView.borderStrokeColor, controller.borderStrokeColor);
+  XCTAssertEqualObjects(textField.borderView.borderStrokeColor, UIColor.redColor);
+}
+
+- (void)testOutlinedTextFieldBorderStrokeColorDefaultsToNormal {
+  // Given
+  MDCTextField *textField = [[MDCTextField alloc] init];
+  MDCTextInputControllerOutlined *controller =
+      [[MDCTextInputControllerOutlined alloc] initWithTextInput:textField];
+  MDCSemanticColorScheme *colorScheme =
+      [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+
+  // When
+  controller.borderStrokeColor = nil;
+  [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme
```

Comment:
let us just set the normal color and check that logic rather than assume that the color themer will do so. By calling the colorThemer we are also assuming that it will not touch the boarderStrokeColor in the future.